maquetado/scss/footer.scss

```diff
@@ -0,0 +1,24 @@
+@import 'colors';
+@import 'variables';
+
+.footer {
+  align-items: center;
+  background-color: $blue;
+  display: flex;
+  flex-direction: column;
+}
+
+.footer-big-text {
+  color: $white;
```

Comment:
Is it necessary assign the color again? If you set the properties `color` and `font-weight` in the class `.footer` isn't necessary put it here